roles/confluent.test/tasks/check_group.yml

```
@@ -0,0 +1,12 @@
+---
+    - name: Check group membership
+      shell: "grep {{full_user}} /etc/group"
+      register: group_member
+      changed_when: false
+    
+    - name: Assert user set as expected
+      assert:
+        that:
+          - expected_value == group_member.stdout.split(':')[0]
```

It appears this test confirms a user is part of a group, why do you need both `group` and `expected_value` as parameters?

shouldn't this be 
```suggestion
          - group == group_member.stdout.split(':')[0]
```